templates/tfengine/recipes/devops.hcl

```diff
@@ -83,6 +97,13 @@ template "cicd_manual" {
   }
 }
 
+{{if get . "enable_terragrunt" true}}
```

Comment:
Just something to think about but instead of enabling or disabling terragrunt we can introduce a variable "terragrunt_root"

In our templates we can set it to `terragrunt_root = "live"`. If that field is missing we don't generate Terragrunt root. It may make it more obvious where the 'live' value is coming from. WDYT?

Reply:
Yea that's a good idea. I like making the value "live" more explicit and obvious. I think it's more like a cicd_root, and terragrunt or terraform can both act on the configs in the root. We can discuss it more in our sync.

Reply:
I still see a cicd in the live dir. Is that supposed to be there?

https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/tree/devops/examples/tfengine/generated/devops/live/cicd

Reply:
Resolved offline. Added a deployment folder.